MQuince/MQuince.WebAPI.Integration.Testing/AppointmentTests.cs

```diff
@@ -79,6 +79,17 @@ public async Task Create_appointments()
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
+        [Fact]
+        public async void Cancel_appointment_status_code_test()
+        {
+            HttpClient client = _factory.CreateClient();
+
+            HttpResponseMessage response = await client.GetAsync("api/appointment/cancelAppointment/08d8a299-ff01-4112-86a8-e5553e4ec81b");
+
+            Assert.False(this.IsOkOrNotFound(response));
+
+        }
+
```

Comment:
In AppointmentController you use PUT method for cancel appointment. In this integration test you use GET and always will return status code 404(Not Found).